faunadb-scala/src/load/scala/faunadb/FaunaClientFixture.scala

```diff
@@ -19,6 +20,7 @@ trait FaunaClientFixture extends SuiteMixin with BeforeAndAfterAll { self: Fixtu
       collection.Map("root_token" -> rootKey, "root_url" -> s"$scheme://$domain:$port")
     }
 
+    _config = config
```

Comment:
Can we also add an _accessor_ for when we need to read the variable? :)

```scala
protected def config: collection.Map[String, String] = _config
```

For instance, when we create the client below for each test:

```scala
FaunaClient(endpoint = config("root_url"), secret = secret)
```